client/src/components/GameSelector/index.js

```diff
@@ -63,6 +60,24 @@ class GameSelector extends Component {
     this.setState({ loading: false });
   }
 
+  getUsername = async uid => {
```

Comment:
Would it be possible to have getUsername as an export somewhere easily accessible by multiple js files? I also grab usernames multiple times while in-game.

Reply:
Is it possible to do that in React? It seems like to have an export you need to render some HTML.

Reply:
Turns out it's pretty easy haha. I just pushed changes!

Reply:
:D